web/frontend/src/api.js

```diff
@@ -7,8 +7,9 @@ export const getApiApplication = async () => {
   const result = await fetch(`${host}/runtimes`)
   const data = await result.json()
 
-  if (data?.length > 0) {
-    const [{ platformaticVersion: pltVersion, packageName: name, pid: id, uptimeSeconds, url }] = data
+  for (const runtime of data) {
```

Comment:
If we make the change mentioned in the [comment above](https://github.com/platformatic/watt-admin/pull/35/files#r1991450808) we can simplify this code (avoiding handling an array of runtimes)

Reply:
I would keep the code as it is right now, we would implement the selector in the UI in the future.